### DISCUSSION TITLE
Hello, have you considered supporting the addition of Alibaba Cloud disk backup? Because Alibaba Cloud disk has no speed limit, if you use Alibaba Cloud disk backup, even if you don't recharge the membership version, the free version space is completely enough.

### DISCUSSION THREAD

**Discussion by dgggh-56** [remotely-save/remotely-save#471] [Ideas]
Hello, have you considered supporting the addition of Alibaba Cloud disk backup? Because Alibaba Cloud disk has no speed limit, if you use Alibaba Cloud disk backup, even if you don't recharge the membership version, the free version space is completely enough.

**Comment by fyears** [comment]
在 #404 讨论
